server/v1/controllers/shows/utils/utils.js

```diff
@@ -18,13 +29,7 @@ function showList(shows, startDate, endDate) {
   const allSeriesShows = reduceShowsByRepeatProperty(shows, true);
 
   const allSeriesShowsExpanded = allSeriesShows.map(show => {
-    const allShowDates = returnDatesArrayByRepeatRule(show, startDate, endDate);
-    const allSeriesShowsExpandedByDates = returnSeriesShowsArrayWithNewDates(
-      allShowDates,
-      show,
-    );
-
-    return allSeriesShowsExpandedByDates;
+    return allShowInstancesInDateRange(show, startDate, endDate); //TODO: ask Josh if there's better naming for this
```

Comment:
Do you know of a better way to name this function? It's calling the exact same code as was removed above (I'm calling this from a place in the playlist model now too, hence the need to make it a function)